deep-learning/_posts/2018-07-05-Ship-wake-detection.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: review
+title:  "Ship wake-detection procedure using conjugate gradient trained artificial neural networks"
+tags:   deep-learning, remote-sensing, neural-network
```

Comment:
remove deep-learning as they only use a 1-layer network.  Also, put your md file in the machine-learning folder